src/App.js

```diff
@@ -1,22 +1,59 @@
-import { FormSetDates } from "./components/Form/FormSetDates";
+import styled from 'styled-components';
 
+import {Navbar} from './components/activity/Navbar';
+import {Homepage} from './components/activity/Hompage';
 
+import {FormAndList} from './FormAndList';
+import {TodoList} from './TodoList';
 
+import {Route, Routes} from 'react-router-dom';
 
 export default function App() {
   return (
-    <>
-    <heading>
-      <h1>Struct your Day</h1>
-    </heading>
-    
-    
-    
-
-    
-  <FormSetDates/>
-  </>
-  
-  )
-  
+    <StyledWrapper>
+      <Navbar />
+      <header>
+        <img width={180} height={180} src="./images/logo.png" alt="logo" />
+        <h1>Structure your Day</h1>
+      </header>
+      <Routes>
+        <Route path="/" element={<Homepage />} />
+        <Route path="/listEntries" element={<FormAndList />} />
+        <Route path="/Calendar" element={<TodoList />} />
```

Comment:
```suggestion
        <Route path="/calendar" element={<TodoList />} />
```
When reading this, I am slightly confused by the naming: What does a calendar has to do with a TodoList?